src/Store/Reducers/SystemsPageStore.js

```diff
@@ -40,6 +40,7 @@ export const SystemsPageStore = (state = initialState, action) => {
         case ActionTypes.CLEAR_SYSTEMS_STORE:
             return {
                 ...initialState,
+                params: { ...newState.params, ...{ page: 1 } },
```

Comment:
This spreading operator is unnecessary
`parameters: { ...state.parameters, page: 1 },`